src/features/modals/CreateToken/CreateTokenBody/TokenSummary/TokenSummary.tsx

```diff
@@ -0,0 +1,97 @@
+//- React Imports
+import React, { FC } from 'react';
+
+//- Style Imports
+import styles from './TokenSummary.module.scss';
+
+//- Component Imports
+import { FormFooter } from '../FormFooter/FormFooter';
+import { TokenSummaryField } from './TokenSummaryField/TokenSummaryField';
+
+import {
+	MediaInput,
+	MediaType,
+} from '@zero-tech/zui/src/components/MediaInput';
+
+const truncateAddress = (address: string): string => {
+	if (address.length < 8) return address;
+	return `${address.substring(0, 4)}...${address.substring(
+		address.length - 4,
+		address.length,
+	)}`;
+};
```

Comment:
This function is [here](https://github.com/zer0-os/zApp-NFTs/blob/main/src/lib/util/domains/domains.ts) - not sure if it was in the repo when you made this so no stress! Change it over just so we don't have the same function defined twice.